src/main/java/org/fundacionjala/pivotal/cucumber/utils/Utils.java

```diff
@@ -69,4 +74,68 @@ public static String buildEndpoint(final String endpoint) {
         return "";
     }
 
+    /**
+     * convert map to json.
+     *
+     * @param body map.
+     * @return element in Json.
+     */
+    public static String convertJson(final Map<String, String> body) {
+        StringBuilder res = new StringBuilder();
+        Iterator it = body.keySet().iterator();
+        res.append("{");
+        while (it.hasNext()) {
+            String key = String.valueOf(it.next());
+            res.append("\"").append(key).append("\"").append(":");
+            if (!body.get(key).contains(",")) {
+                System.out.println(body.get(key));
+                res.append("\"").append(body.get(key)).append("\"");
+
+            } else {
+                res.append("[").append(dataValue(body.get(key))).append("]");
+            }
+            res.append(it.hasNext() ? "," : "");
+
+        }
+        res.append("}");
+        return res.toString();
+    }
+
+    /**
+     * draw value of String.
+     *
+     * @param array String where is the value.
+     * @return string of value.
+     */
+    private static String dataValue(final String array) {
+
+        String[] point = array.split(",");
+        for (int i = 0; i < point.length; i++) {
+            System.out.println(point[i]);
+            point[i] = buildEndpoint(point[i]);
+            System.out.println(point[i]);
+        }
+        return String.join(",", point);
+
+    }
+
+    /**
+     * verify if had , in.
+     *
+     * @param body map where search.
+     * @return boolean.
+     */
+    public static boolean verifyArray(final Map<String, String> body) {
```

Comment:
This method can be refactor in a single line of code using stream.
@AT-03/at-03 
Please review it.